### PR TITLE
Applied round shape on profile image

### LINKED ISSUE #2715: Styling profile picture in search page and user dashboard
**Is your feature request related to a problem? Please describe.**
The way Profile pictures are displayed is not appealing in case they are not round or square shaped. Most websites use round border to show profile pictures. 

**Describe the solution you'd like**
I will add a new class ```"search-image-cropper"``` to the div that contains the profile picture which will give the image its round shape. Also, I will further tweak the css of class ```"search-usersearch-image"``` 

**Screen Shots**
### User Dashboard

| Before      | After          |
| ------------- |:-------------:|
| before      | after |

### Search Page

| Before      | After          |
| ------------- |:-------------:|
| before-search| after-search|

**Are you working on this? (Yes)**
Yes, I have done the required changes.

### PR DESCRIPTION
**Pull request by camperjett**
Fixes #2715 

#### Describe the changes you have made in this PR -
In index.html, I added a class ```"search-image-cropper"``` to the div containing the user image.
In search.html, I did the same.
In search.scss, I tweaked the class ```"search-usersearch-image"``` as well as added a new class ```"search-image-cropper"```
### Screenshots of the changes (If any) -

**Screen Shots**
### User Dashboard

| Before      | After          |
| ------------- |:-------------:|
| before      | after |

### Search Page

| Before      | After          |
| ------------- |:-------------:|
| before-search| after-search|

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Review by vedant-jain03 — APPROVED** [review]

**Comment by camperjett** [comment]
Do we need any changes? @vedant-jain03

**Comment by vedant-jain03** [comment]
> Do we need any changes? @vedant-jain03

Wait for @nitin10s review.

**Review by nitin10s — COMMENTED** [review]
I think the current designs are apt for the time being and the new designs suggested are marginal improvements. Before making a PR, please get your issues approved by a maintainer.

**Pull request closed without merging**
